src/js/base/model.js

```diff
@@ -98,12 +109,30 @@ export default Backbone.Model.extend(extend({
 
     return isFunction(messages[category]) ? messages[category] : this[messages[category]];
   },
-  handleMessage({ category, resource, author, payload }) {
-    payload.attributes = extend({}, payload.attributes, { updated_at: dayjs.utc().format() });
+  queueMessage(data) {
```

Comment:
So this function looks complex to me, but I get why.. but we could start using some of the newer operators to make this a bit simpler:

```js
queueMessage(data) {
 return (this._notifications ||= {})[data.category] = data;
}
```

Reply:
That's pretty cool, didn't know `||=` existed until now 👍 

But `vite-plugin-eslint` doesn't support it. It's using eslint `v8.21.0` ([link](https://github.com/gxmari007/vite-plugin-eslint/blob/main/package.json#L50)). Eslint added support logical assignment operators in `v8.24.0` ([source](https://eslint.org/docs/latest/rules/logical-assignment-operators#version)).

That package's repo seems to be abandoned, so we probably want to switch to something different.

Reply:
That's just the dev dependency.  We're using 9 and it's a peer dependency https://github.com/gxmari007/vite-plugin-eslint/blob/main/package.json#L60

Reply:
FWIW it looks like the vite community doesn't run eslint as part of the build process.  They're using it in pre-commit pre-build hooks..  I find that mildly annoying

Reply:
FWIW backend runs static analysis (including lint) on commit and on test (CI).